README.md

```diff
@@ -1,5 +1,34 @@
-A tool to dump the ROM of the Xbox DVD Movie Playback Kit dongle.
+## A tool to dump the ROM of the Xbox DVD Movie Playback Kit dongle.
 
 The ROM is prefixed with version information and the length of the contained XBE.
 The XBE certificate region is the allowed DVD playback region.
 The section of the XBE is compressed using LZX, 128kiB window size, 32kiB blocks.
+
+## Prerequisites
+
+Make sure you satisfy the following requirements:
+
+* basic python know-how
+* pipenv
+* libusb1
+* a way to attach the DVD Kit dongle to your computer i.e. this [[adapter]](https://www.amazon.ca/Mcbazel-Female-Controller-Adapter-Cable/dp/B000RT2868).
```

Comment:
GitHub markdown should be using `[link description]` and not `[[link description]]`?